docs/Local_Development_Tutorial.md

```diff
@@ -0,0 +1,168 @@
+- Set up the prerequisites necessary for local development using MarkLogic Server in Kubernetes 
+- How to run Minikube and load MarkLogic Server into the Kubernetes cluster 
+- Access the MarkLogic Server cluster
+- How to debug the Kubernetes environment 
+- How to clean up your environment
+
+
+## Prerequisites
+The following steps assume you are running this tutorial from a desktop environment. Mobile environments will likely experience problems and may not work.
+- [Docker](https://docs.docker.com/engine/install/): Subscribe to Docker Hub and pulldown the latest image from: https://hub.docker.com/_/marklogic
+  ```sh
+  # Something similar to this, with the latest version tag, which can be found on the dockerhub link above
+  docker pull store/marklogicdb/marklogic-server:10.0-9-centos-1.0.0-ea4 
+  ```
+- [KubeCTL](https://kubernetes.io/docs/tasks/tools/):  Download and install this tool to assist with debugging in a Kubernetes environment. 
+- [Helm](https://helm.sh/docs/intro/install/):  Clone or download the chart repository: https://github.com/marklogic/marklogic-kubernetes
+- [Minikube](https://k8s-docs.netlify.app/en/docs/tasks/tools/install-minikube/): Minikube Kubernetes environment, which will host the MarkLogic Server application.
+- Mac OSX VM: If you are running in Mac OSX, you will need Virtual Box: [Virtual Box](https://www.virtualbox.org/) the VM used inside of Minikube. 
+- Browser: The latest version of a supported web browser. See the list here: [Web Browser](https://developer.marklogic.com/products/support-matrix/) 
+
+
+# Procedure 
+This section describes the procedure for setting up Minikube, installing MarkLogic Server, and verifying the installation. 
+
+
+## Setting Up Minikube
+First you will need to set up the Kubernetes control plane on your local machine. Minikube is a tool that makes it easy to set up a local Kubernetes enviornment.
+
+- Start Minikube: `minikube start --driver=virtualbox`
+ If you are running in a Linux OS: `minikube start --driver=docker`  
+
+To verify the Minikube started correctly, use the Kubernetes command line tool, KubeCTL:  
+
+```sh
+kubectl get nodes
+```
+```
+Expected Output: 
+NAME       STATUS   ROLES                  AGE   VERSION
+minikube   Ready    control-plane,master   1d    v1.23.3
+```
+
+ To enable addons run the follow Minikube command for ingress:     
+ `minikube addons enable ingress`  
+##  Installing a Single MarkLogic Host to Minikube
+- Push the image used for MarkLogic Server to the VM:   
+`minikube image load store/marklogicdb/marklogic-server:10.0-9-centos-1.0.0-ea4`  
+  The image ID used in the example is `store/marklogicdb/marklogic-server:10.0-9-centos-1.0.0-ea4`, which may not be the latest image. To find the latest ID go to https://hub.docker.com/_/marklogic  
+- Navigate to where you downloaded or cloned the MarkLogic Helm repository.
```

Comment:
For internal developer like us, we can install helm chart directly from the repo. For customers, we probably want them to install from our office site. Then do the following:
1. `helm repo add marklogic https://github.com/marklogic/marklogic-kubernetes`
2. `helm install my-release marklogic/marklogic --version=1.0.0-ea1`
since my code haven't merge to master yet, to try out the first step you can use my repo for testing purpose
`helm repo add marklogic https://pengzhouml.github.io/marklogic-kubernetes/`
To verify, use `helm repo list`
you

Reply:
Updated: 201b430